README.md

```diff
@@ -113,6 +107,64 @@ Finally, properties are what give the selected elements of a rule declaration th
   - Uses of z-index
   - Compatibility or browser-specific hacks
 
+* Prefer line comments (`//` in Sass-land) to block comments.
+* Prefer comments on their own line. Avoid end-of-line comments.
+* Write detailed comments for code that isn't self-documenting:
+  - Uses of z-index
+  - Compatibility or browser-specific hacks
+
+### Colors
+
+When implementing feature styles, you should only be using color variables provided by theme.css
+
+When adding a color variable to theme.css, only use colors available to you by
+the color palette. Those should be listed to you in a separate file called
+`color_palette.css`.
+
+Using HEX color units is preferred over RGB, RGBA, named, HSL, or HSLA values.
+
+**Bad**
+
+```css
+rgb(50, 50, 50);
+white;
+hsl(120, 100%, 50%);
+hsla(120, 100%, 50%, 1);
+```
+
+**Good**
+
+```css
+#FFF;
+#FFFFFF;
+```
+
+Using RGBA to be able to set opacity is not a big problem, but has the downside
```

Comment:
I might change this to "in order to be able" or just cutting the "to be able" entirely